package.json

```diff
@@ -18,45 +18,52 @@
     }
   ],
   "scripts": {
```

Comment:
The dist folder is cleaned by the [clean-webpack-plugin](https://github.com/roots/sage/blob/sage-9/webpack.config.js#L95)